src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,73 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String[] TYPE_OPERATION = {"supply", "buy"};
+    private static final int POSITION_OPERATION_IN_LINE = 0;
+    private static final int POSITION_COST_IN_LINE = 1;
+    private static final int POSITION_ALL_SUPPLY_OPERATION = 0;
+    private static final int POSITION_All_BUY_OPERATION = 1;
+
+    private void getWriteToFile(String lineOperation, String toFileName, int resultSum) {
+        try (BufferedWriter bufferedWriter =
+                     new BufferedWriter(new FileWriter(toFileName, true))) {
+            StringBuilder dataLineToFile = new StringBuilder();
+            bufferedWriter.write(dataLineToFile.append(lineOperation)
+                    .append(",")
+                    .append(resultSum)
+                    .append("\r\n")
+                    .toString());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data file " + toFileName, e);
+        }
+    }
+
+    private int getReadAndSumOperation(String fromFileName, String lineOperation) {
+        String line;
+        int sumOperation;
+        String dataLineFromFile;
+        String[] dataLineSplit;
```

Comment:
let's declare variables at time of their first usage, if possible
it will make the code more readable